igcollect/strongswan.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""igcollect - Strongswan stats
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import re
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='strongswan')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    template = args.prefix + '.{} {} ' + str(int(time()))
+    print(template.format('clients', count_connected_clients()))
```

Comment:
If `clients` is fixed, we could also already put it in the template.

Reply:
I expect more stats to be added in future.